src/core/src/op/subtract.cpp

```diff
@@ -19,14 +19,11 @@ struct Evaluate : element::NoAction<bool> {
     static result_type visit(const Tensor& in0,
                              const Tensor& in1,
                              Tensor& out,
+                             const Shape& shape0,
+                             const Shape& shape1,
```

Comment:
why additional arguments required? Is it possible to call `visit` with `in0.get_shape() != shape0`?

Reply:
Is not required, but changed as it optimized better when shapes are passed as parameters instead doing `get_shape` inside template. 
This is operator's internal implementation only for operator and correct shapes are taken from tensors.